cnf-certification-test/observability/suite.go

```diff
@@ -124,3 +129,19 @@ func testCrds(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("%d CRDs don't have status subresource", n))
 	}
 }
+
+// testTerminationMessagePolicy tests to make sure that pods
+func testTerminationMessagePolicy(env *provider.TestEnvironment) {
+	failedContainers := []string{}
+	for _, cut := range env.Containers {
+		ginkgo.By("Testing pod " + cut.Podname + " and container: " + cut.Data.Name + " for terminationMessagePolicy")
+		if cut.Data.TerminationMessagePolicy != v1.TerminationMessageFallbackToLogsOnError {
+			tnf.ClaimFilePrintf("FAILURE: Pod %s container %s does not have a TerminationMessagePolicy: FallbackToLogsOnError")
+			failedContainers = append(failedContainers, cut.Data.Name)
+		}
+	}
+	if n := len(failedContainers); n > 0 {
+		logrus.Debugf("There are %d containers that do not have a termination message policy set to FallbackToLogsOnError: %+v", len(failedContainers), failedContainers)
```

Comment:
Maybe we can skip printing the list of containers here since there is not pod/name space info and there might be duplicate. the list of failed containers is already printed above